### PR TITLE
5.7.37 psmerge part 1

### PR DESCRIPTION
**Pull request by nitendra-bhosle**

### CODE REVIEW

**Comment by nitendra-bhosle** [comment]
Jenkins : https://pxc.cd.percona.com/view/PXC%205.7/job/pxc-5.7-param/283/

**Comment by nitendra-bhosle** [comment]
Updated jenkins : https://pxc.cd.percona.com/view/PXC%205.7/job/pxc-5.7-param/287/

File: sql/rpl_mi.cc [hunk 1]
```diff
@@ -404,7 +404,7 @@ bool Master_info::read_info(Rpl_info_handler *from)
   if (from->get_info(&temp_master_log_pos,
                      (ulong) BIN_LOG_HEADER_SIZE) ||
       from->get_info(host, sizeof(host), (char *) 0) ||
-      from->get_info(user, sizeof(user), (char *) 0) ||
+      from->get_info(user, sizeof(user), (char *) "test") ||
```
Comment (venkatesh-prasad-v) [inline sql/rpl_mi.cc]: This change re-introduces the bug that was fixed in commit efd3968b98fd6affe5b582da34f4a7fc95c546e0.
Can you please revert this change and check if something can be fixed in test itself?

**Review by venkatesh-prasad-v — CHANGES_REQUESTED** [review]

**Review by venkatesh-prasad-v — APPROVED** [review]
LGTM

**Review by venkatesh-prasad-v — APPROVED** [review]

**Review by kamil-holubicki — APPROVED** [review]
lgtm

**Pull request merged**